lib/newman_conway.py

```
@@ -2,9 +2,28 @@
 
 # Time complexity: ?
 # Space Complexity: ?
-def newman_conway(num):
+def newman_conway(nums):
     """ Returns a list of the Newman Conway numbers for the given value.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
+        Space Complexity: O(n^2)
     """
-    pass
+    if nums == 0:
+        raise ValueError
```

We should raise this error for _any_ value below the valid starting point of the sequence:
```py
    if nums <= 0:
        raise ValueError
```